### DISCUSSION TITLE
Some triggers not firing

### DISCUSSION THREAD

**Discussion by jward23** [koenbeuk/EntityFrameworkCore.Triggered#202] [Q&A]
We have recently started using EFCore.Triggered and have ben enjoying it's use.  However for some reason the triggers do not fire for a couple of our entities.  No error, they are just never called.  They are registered in the startup and we have several other triggers there that work fine.  In your experience have you ever run into such a situation?  Is there any typical issues in the entity or service that might prevent the trigger from executing?  Thank You.

**Comment by rlabrador** [comment]
Is AsNoTracking() flag enabled  on those entities? the use of UseQueryTrackingBehavior?
